patches/0103-Adjust-Go-tests-to-work-with-crypto-module.patch

```diff
@@ -3,12 +3,50 @@ From: microsoft-golang-bot <microsoft-golang-bot@users.noreply.github.com>
 Date: Thu, 31 Mar 2022 13:11:06 -0500
 Subject: [PATCH] Adjust Go tests to work with crypto module
 
+use gocrypt tag for cgo static tests
```

Comment:
Hmm. Not sure whether we should include an overall summary of each change here, since this file contains some loosely related changes. For most changes, we can have code comments that describe the change in-place, which is great. But others, like the change in the middle of a string in `src/go/build/deps_test.go` can't really have comments all that nearby, so maybe those need more description in the summary--but then it's hard to know which description describes which changes.

I think test changes probably actually deserve their own patch files, so the patch summary can be more tightly linked to specific changes. This also makes it easier to remove changes where we're changed various parts of Go due to a single upstream issue: just remove the whole individual patch file once the issue is fixed, rather than searching for references to the issue in a single patch.

Reply:
Agree, but as you also proposed, let's fix this in another PR so we can start testing FIPS mode.